### DISCUSSION TITLE
Roborock S8 Pro Ultra

### DISCUSSION THREAD

**Discussion by weijiethe188** [humbertogontijo/homeassistant-roborock#363] [Q&A]
Has anyone tried the integration with Roborock S8? Does it work well?

**Comment by Lash-L** [comment]
I'm pretty sure we have a good bit of users with S8's. Their main complaint is that we don't support most docking functionality, but other than that they seem happy

**Reply by lxto** [reply]
we have a fine working S8 at home. After some research we manged to install all funktionality incl. "go-to" and "spot-cleaning". Ver.1.0.8 even works fine after issues with Ver.1.0.7
